.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think you might want to add `.vscode` to your `.gitignore` file. 
IDE/Code editor specific files should be kept only locally, please remove this file from the pull request :)